.vscodeignore

```diff
@@ -0,0 +1,4 @@
+**/*
+!out/*.js
+!LICENSE
+!README.md
```

Comment:
Publish to vscode marketplace only builded files, readme and license (not sure about license)